Sources/StreamChat/Repositories/AuthenticationRepository.swift

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright © 2022 Stream.io Inc. All rights reserved.
+// Copyright © 2023 Stream.io Inc. All rights reserved.
```

Comment:
This will probably lead to a merge conflict in 4.26.0 🤔

Reply:
Good catch!